aries-site/src/examples/components/namevaluelist/Previews/NameValueListAnatomyPreview.js

```diff
@@ -1,19 +1,30 @@
-import React from 'react';
-import { Anchor, Box, NameValueList, NameValuePair } from 'grommet';
+import React, { useContext } from 'react';
+import {
+  Anchor,
+  Box,
+  NameValueList,
+  NameValuePair,
+  ResponsiveContext,
+} from 'grommet';
 import { anatomyData } from '../data';
 
-export const NameValueListAnatomyPreview = () => (
-  <Box pad="small">
-    <NameValueList>
-      {Object.entries(anatomyData).map(([name, value]) => (
-        <NameValuePair key={name} name={name}>
-          {name === 'Created by' ? (
-            <Anchor label={value} href={`mailto:${value}`} />
-          ) : (
-            <>{value}</>
-          )}
-        </NameValuePair>
-      ))}
-    </NameValueList>
-  </Box>
-);
+export const NameValueListAnatomyPreview = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box pad="small">
+      <NameValueList
+        valueProps={size === 'medium' ? { width: 'small' } : undefined}
```

Comment:
I think this works for now, but I almost wonder if NameValueList should wrap to `column` when there's not enough space for it in the row?

Reply:
In grommet your referring to?

Reply:
Yeah, like a future enhancement (not for right now)

Reply:
yes I agree we can relook at how the columns are responsive

Reply:
What is instead of this we say:

```
<NameValueList
      pairProps={{ direction: size === 'medium' || size === 'small' ? 'column' : 'row'}}
...
```

This way we are keeping with our typical widths and guidance but just adjusting the responsive behavior to what we might lean towards in the future.

Reply:
that's a good point.. we dont really want to show changing the `widtth`